lib/repl.js

```diff
@@ -602,7 +602,19 @@ REPLServer.prototype.complete = function(line, callback) {
           // if (e) console.log(e);
 
           if (obj != null) {
-            if (util.isObject(obj) || util.isFunction(obj)) {
+            if (Array.isArray(obj)) {
+              // Array elements will contain many integer values
+              // if we return them in memberGroups they will appear
+              // as obj.0 obj.100 etc. which is not valid syntax.
+              // Additionally for large arrays this can make for a
+              // very cluttered list.  So I filter them out.
```

Comment:
heh, yes, I completely ignored the comments in the original patch.. I'll edit those.